[2024-10-21 20:22:02,149: INFO: main: Welcome to custom log.]
[2024-10-24 13:58:02,664: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-10-24 13:58:02,667: INFO: common: yaml file: params.yaml loaded successfully]
[2024-10-24 13:58:02,670: INFO: common: created directory at: artifacts]
[2024-10-24 13:58:02,672: INFO: common: created directory at: artifacts/data_ingestion]
[2024-10-24 13:58:02,675: INFO: 986331348: Downloading data from https://drive.google.com/file/d/1Q0WzSl389ofCKy_AX0UjFnQwB73OW-xU/view?usp=drive_link into file artifacts/data_ingestion/data.zip]
[2024-10-24 13:58:40,284: INFO: 986331348: Downloaded data from https://drive.google.com/file/d/1Q0WzSl389ofCKy_AX0UjFnQwB73OW-xU/view?usp=drive_link into file artifacts/data_ingestion/data.zip]
[2024-10-24 18:24:46,823: INFO: main: >>>>>> stage Data Ingestion Stage started <<<<<<]
[2024-10-24 18:24:46,825: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-10-24 18:24:46,826: INFO: common: yaml file: params.yaml loaded successfully]
[2024-10-24 18:24:46,827: INFO: common: created directory at: artifacts]
[2024-10-24 18:24:46,827: INFO: common: created directory at: artifacts/data_ingestion]
[2024-10-24 18:24:46,828: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1Q0WzSl389ofCKy_AX0UjFnQwB73OW-xU/view?usp=drive_link into file artifacts/data_ingestion/data.zip]
[2024-10-24 18:25:19,225: INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/1Q0WzSl389ofCKy_AX0UjFnQwB73OW-xU/view?usp=drive_link into file artifacts/data_ingestion/data.zip]
[2024-10-24 18:25:20,247: INFO: main: >>>>>> stage Data Ingestion Stage completed <<<<<<

x======x]
[2024-10-25 15:24:44,512: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-10-25 15:24:44,546: INFO: common: yaml file: params.yaml loaded successfully]
[2024-10-25 15:24:44,557: INFO: common: created directory at: artifacts]
[2024-10-25 17:56:44,765: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-10-25 17:56:44,771: INFO: common: yaml file: params.yaml loaded successfully]
[2024-10-25 17:56:44,773: INFO: common: created directory at: artifacts]
[2024-10-25 17:56:44,776: INFO: common: created directory at: artifacts/prepare_base_model]
[2024-10-25 17:57:02,663: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-10-26 01:26:54,930: INFO: main: >>>>>> stage Data Ingestion Stage started <<<<<<]
[2024-10-26 01:26:54,933: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-10-26 01:26:54,951: INFO: common: yaml file: params.yaml loaded successfully]
[2024-10-26 01:26:54,952: INFO: common: created directory at: artifacts]
[2024-10-26 01:26:54,952: INFO: common: created directory at: artifacts/data_ingestion]
[2024-10-26 01:26:54,953: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1Q0WzSl389ofCKy_AX0UjFnQwB73OW-xU/view?usp=drive_link into file artifacts/data_ingestion/data.zip]
[2024-10-26 01:27:31,884: INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/1Q0WzSl389ofCKy_AX0UjFnQwB73OW-xU/view?usp=drive_link into file artifacts/data_ingestion/data.zip]
[2024-10-26 01:27:34,853: INFO: main: >>>>>> stage Data Ingestion Stage completed <<<<<<

x======x]
[2024-10-26 01:29:03,196: INFO: main: >>>>>> stage Data Ingestion Stage started <<<<<<]
[2024-10-26 01:29:03,199: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-10-26 01:29:03,201: INFO: common: yaml file: params.yaml loaded successfully]
[2024-10-26 01:29:03,201: INFO: common: created directory at: artifacts]
[2024-10-26 01:29:03,201: INFO: common: created directory at: artifacts/data_ingestion]
[2024-10-26 01:29:03,202: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1Q0WzSl389ofCKy_AX0UjFnQwB73OW-xU/view?usp=drive_link into file artifacts/data_ingestion/data.zip]
[2024-10-26 01:29:46,514: INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/1Q0WzSl389ofCKy_AX0UjFnQwB73OW-xU/view?usp=drive_link into file artifacts/data_ingestion/data.zip]
[2024-10-26 01:29:47,531: INFO: main: >>>>>> stage Data Ingestion Stage completed <<<<<<

x======x]
[2024-10-26 01:29:47,532: INFO: main: >>>>>> stage Prepare base model started <<<<<<]
[2024-10-26 01:29:47,535: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-10-26 01:29:47,538: INFO: common: yaml file: params.yaml loaded successfully]
[2024-10-26 01:29:47,539: INFO: common: created directory at: artifacts]
[2024-10-26 01:29:47,540: INFO: common: created directory at: artifacts/prepare_base_model]
[2024-10-26 01:29:48,066: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-10-26 01:29:48,423: INFO: main: >>>>>> stage Prepare base model completed <<<<<<

x======x]
[2024-10-26 02:05:06,544: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-10-26 02:05:06,555: INFO: common: yaml file: params.yaml loaded successfully]
[2024-10-26 02:05:06,557: INFO: common: created directory at: artifacts]
[2024-10-26 02:05:06,561: INFO: common: created directory at: artifacts\training]
[2024-10-26 02:08:11,661: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-10-26 02:08:11,665: INFO: common: yaml file: params.yaml loaded successfully]
[2024-10-26 02:08:11,667: INFO: common: created directory at: artifacts]
[2024-10-26 02:08:11,670: INFO: common: created directory at: artifacts\training]
[2024-10-26 02:11:11,890: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-10-26 02:11:11,903: INFO: common: yaml file: params.yaml loaded successfully]
[2024-10-26 02:11:11,905: INFO: common: created directory at: artifacts]
[2024-10-26 02:11:11,907: INFO: common: created directory at: artifacts\training]
[2024-10-26 02:13:36,709: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-10-26 02:13:36,714: INFO: common: yaml file: params.yaml loaded successfully]
[2024-10-26 02:13:36,716: INFO: common: created directory at: artifacts]
[2024-10-26 02:13:36,718: INFO: common: created directory at: artifacts\training]
[2024-10-26 02:43:50,406: INFO: main: >>>>>> stage Data Ingestion Stage started <<<<<<]
[2024-10-26 02:43:50,409: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-10-26 02:43:50,411: INFO: common: yaml file: params.yaml loaded successfully]
[2024-10-26 02:43:50,412: INFO: common: created directory at: artifacts]
[2024-10-26 02:43:50,412: INFO: common: created directory at: artifacts/data_ingestion]
[2024-10-26 02:43:50,412: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1Q0WzSl389ofCKy_AX0UjFnQwB73OW-xU/view?usp=drive_link into file artifacts/data_ingestion/data.zip]
[2024-10-26 02:44:22,823: INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/1Q0WzSl389ofCKy_AX0UjFnQwB73OW-xU/view?usp=drive_link into file artifacts/data_ingestion/data.zip]
[2024-10-26 02:44:23,976: INFO: main: >>>>>> stage Data Ingestion Stage completed <<<<<<

x======x]
[2024-10-26 02:44:23,977: INFO: main: >>>>>> stage Prepare base model started <<<<<<]
[2024-10-26 02:44:23,980: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-10-26 02:44:23,982: INFO: common: yaml file: params.yaml loaded successfully]
[2024-10-26 02:44:23,982: INFO: common: created directory at: artifacts]
[2024-10-26 02:44:23,983: INFO: common: created directory at: artifacts/prepare_base_model]
[2024-10-26 02:44:24,598: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-10-26 02:44:25,017: INFO: main: >>>>>> stage Prepare base model completed <<<<<<

x======x]
[2024-10-26 02:44:25,018: INFO: main: >>>>>> stage Training started <<<<<<]
[2024-10-26 02:44:25,021: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-10-26 02:44:25,024: INFO: common: yaml file: params.yaml loaded successfully]
[2024-10-26 02:44:25,025: INFO: common: created directory at: artifacts]
[2024-10-26 02:44:25,026: ERROR: main: 'ConfigurationManager' object has no attribute 'get_training_config']
Traceback (most recent call last):
  File "d:/FULL Projects/Kidney-Disease-Classification/main.py", line 41, in <module>
    obj.main()
  File "d:\full projects\kidney-disease-classification\src\cnnClassifier\pipeline\stage_03_model_training.py", line 13, in main
    training_config = config.get_training_config()
AttributeError: 'ConfigurationManager' object has no attribute 'get_training_config'
[2024-10-26 02:46:24,189: INFO: main: >>>>>> stage Data Ingestion Stage started <<<<<<]
[2024-10-26 02:46:24,192: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-10-26 02:46:24,194: INFO: common: yaml file: params.yaml loaded successfully]
[2024-10-26 02:46:24,195: INFO: common: created directory at: artifacts]
[2024-10-26 02:46:24,195: INFO: common: created directory at: artifacts/data_ingestion]
[2024-10-26 02:46:24,196: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1Q0WzSl389ofCKy_AX0UjFnQwB73OW-xU/view?usp=drive_link into file artifacts/data_ingestion/data.zip]
[2024-10-26 02:46:56,391: INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/1Q0WzSl389ofCKy_AX0UjFnQwB73OW-xU/view?usp=drive_link into file artifacts/data_ingestion/data.zip]
[2024-10-26 02:46:57,535: INFO: main: >>>>>> stage Data Ingestion Stage completed <<<<<<

x======x]
[2024-10-26 02:46:57,535: INFO: main: >>>>>> stage Prepare base model started <<<<<<]
[2024-10-26 02:46:57,539: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-10-26 02:46:57,541: INFO: common: yaml file: params.yaml loaded successfully]
[2024-10-26 02:46:57,542: INFO: common: created directory at: artifacts]
[2024-10-26 02:46:57,543: INFO: common: created directory at: artifacts/prepare_base_model]
[2024-10-26 02:46:58,161: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-10-26 02:46:58,483: INFO: main: >>>>>> stage Prepare base model completed <<<<<<

x======x]
[2024-10-26 02:46:58,484: INFO: main: >>>>>> stage Training started <<<<<<]
[2024-10-26 02:46:58,488: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-10-26 02:46:58,490: INFO: common: yaml file: params.yaml loaded successfully]
[2024-10-26 02:46:58,491: INFO: common: created directory at: artifacts]
[2024-10-26 02:46:58,491: ERROR: main: 'ConfigurationManager' object has no attribute 'get_training_config']
Traceback (most recent call last):
  File "d:/FULL Projects/Kidney-Disease-Classification/main.py", line 41, in <module>
    obj.main()
  File "d:\full projects\kidney-disease-classification\src\cnnClassifier\pipeline\stage_03_model_training.py", line 13, in main
    training_config = config.get_training_config()
AttributeError: 'ConfigurationManager' object has no attribute 'get_training_config'
[2024-10-26 02:51:23,391: INFO: main: >>>>>> stage Data Ingestion Stage started <<<<<<]
[2024-10-26 02:51:23,394: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-10-26 02:51:23,396: INFO: common: yaml file: params.yaml loaded successfully]
[2024-10-26 02:51:23,396: INFO: common: created directory at: artifacts]
[2024-10-26 02:51:23,397: INFO: common: created directory at: artifacts/data_ingestion]
[2024-10-26 02:51:23,397: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1Q0WzSl389ofCKy_AX0UjFnQwB73OW-xU/view?usp=drive_link into file artifacts/data_ingestion/data.zip]
[2024-10-26 02:51:55,868: INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/1Q0WzSl389ofCKy_AX0UjFnQwB73OW-xU/view?usp=drive_link into file artifacts/data_ingestion/data.zip]
[2024-10-26 02:51:56,861: INFO: main: >>>>>> stage Data Ingestion Stage completed <<<<<<

x======x]
[2024-10-26 02:51:56,861: INFO: main: >>>>>> stage Prepare base model started <<<<<<]
[2024-10-26 02:51:56,864: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-10-26 02:51:56,866: INFO: common: yaml file: params.yaml loaded successfully]
[2024-10-26 02:51:56,867: INFO: common: created directory at: artifacts]
[2024-10-26 02:51:56,867: INFO: common: created directory at: artifacts/prepare_base_model]
[2024-10-26 02:51:57,389: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-10-26 02:51:57,759: INFO: main: >>>>>> stage Prepare base model completed <<<<<<

x======x]
[2024-10-26 02:51:57,760: INFO: main: >>>>>> stage Training started <<<<<<]
[2024-10-26 02:51:57,763: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-10-26 02:51:57,765: INFO: common: yaml file: params.yaml loaded successfully]
[2024-10-26 02:51:57,766: INFO: common: created directory at: artifacts]
[2024-10-26 02:51:57,767: INFO: common: created directory at: artifacts\training]
[2024-10-26 02:54:32,245: INFO: main: >>>>>> stage Training completed <<<<<<

x======x]
[2024-10-26 14:27:10,066: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-10-26 14:27:10,093: INFO: common: yaml file: params.yaml loaded successfully]
[2024-10-26 14:27:10,098: INFO: common: created directory at: artifacts]
[2024-10-26 14:28:31,111: INFO: common: json file saved at: scores.json]
[2024-10-26 14:28:50,594: WARNING: connectionpool: Retrying (Retry(total=4, connect=5, read=4, redirect=5, status=5)) after connection broken by 'ProtocolError('Connection aborted.', ConnectionResetError(10054, 'An existing connection was forcibly closed by the remote host', None, 10054, None))': /Luminousbyte/Kidney-Disease-Classification.mlflow/api/2.0/mlflow/runs/create]
[2024-10-26 14:28:58,314: WARNING: save: Found untraced functions such as _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op while saving (showing 5 of 14). These functions will not be directly callable after loading.]
[2024-10-26 14:28:59,207: INFO: builder_impl: Assets written to: C:\Users\Pinaki\AppData\Local\Temp\tmpdfr_z6id\model\data\model\assets]
[2024-10-27 12:02:11,020: INFO: main: >>>>>> stage Data Ingestion Stage started <<<<<<]
[2024-10-27 12:02:11,055: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-10-27 12:02:11,088: INFO: common: yaml file: params.yaml loaded successfully]
[2024-10-27 12:02:11,090: INFO: common: created directory at: artifacts]
[2024-10-27 12:02:11,095: INFO: common: created directory at: artifacts/data_ingestion]
[2024-10-27 12:02:11,098: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1Q0WzSl389ofCKy_AX0UjFnQwB73OW-xU/view?usp=drive_link into file artifacts/data_ingestion/data.zip]
[2024-10-27 12:02:46,855: INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/1Q0WzSl389ofCKy_AX0UjFnQwB73OW-xU/view?usp=drive_link into file artifacts/data_ingestion/data.zip]
[2024-10-27 12:02:48,078: INFO: main: >>>>>> stage Data Ingestion Stage completed <<<<<<

x======x]
[2024-10-27 12:02:48,079: INFO: main: >>>>>> stage Prepare base model started <<<<<<]
[2024-10-27 12:02:48,087: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-10-27 12:02:48,091: INFO: common: yaml file: params.yaml loaded successfully]
[2024-10-27 12:02:48,092: INFO: common: created directory at: artifacts]
[2024-10-27 12:02:48,093: INFO: common: created directory at: artifacts/prepare_base_model]
[2024-10-27 12:02:48,884: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-10-27 12:02:49,183: INFO: main: >>>>>> stage Prepare base model completed <<<<<<

x======x]
[2024-10-27 12:02:49,183: INFO: main: >>>>>> stage Training started <<<<<<]
[2024-10-27 12:02:49,186: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-10-27 12:02:49,189: INFO: common: yaml file: params.yaml loaded successfully]
[2024-10-27 12:02:49,189: INFO: common: created directory at: artifacts]
[2024-10-27 12:02:49,190: INFO: common: created directory at: artifacts\training]
[2024-10-27 12:06:00,307: INFO: main: >>>>>> stage Training completed <<<<<<

x======x]
[2024-10-27 12:06:00,309: INFO: main: *******************]
[2024-10-27 12:06:00,309: INFO: main: >>>>>> stage Evaluation Stage started <<<<<<]
[2024-10-27 12:06:00,314: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-10-27 12:06:00,317: INFO: common: yaml file: params.yaml loaded successfully]
[2024-10-27 12:06:00,318: INFO: common: created directory at: artifacts]
[2024-10-27 12:06:56,155: INFO: common: json file saved at: scores.json]
[2024-10-27 12:07:01,942: WARNING: save: Found untraced functions such as _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op while saving (showing 5 of 14). These functions will not be directly callable after loading.]
[2024-10-27 12:07:02,956: INFO: builder_impl: Assets written to: C:\Users\Pinaki\AppData\Local\Temp\tmpbqe0k0nr\model\data\model\assets]
[2024-10-27 12:07:24,132: ERROR: main: No module named 'distutils._modified']
Traceback (most recent call last):
  File "d:/FULL Projects/Kidney-Disease-Classification/main.py", line 54, in <module>
    obj.main()
  File "d:\full projects\kidney-disease-classification\src\cnnClassifier\pipeline\stage_04_model_evaluation.py", line 16, in main
    evaluation.log_into_mlflow()
  File "d:\full projects\kidney-disease-classification\src\cnnClassifier\components\model_evaluation_mlflow.py", line 73, in log_into_mlflow
    mlflow.keras.log_model(self.model, "model")
  File "C:\Users\Pinaki\anaconda3\envs\kidney\lib\site-packages\mlflow\tensorflow\__init__.py", line 222, in log_model
    return Model.log(
  File "C:\Users\Pinaki\anaconda3\envs\kidney\lib\site-packages\mlflow\models\model.py", line 552, in log
    mlflow.tracking.fluent.log_artifacts(local_path, mlflow_model.artifact_path)
  File "C:\Users\Pinaki\anaconda3\envs\kidney\lib\site-packages\mlflow\tracking\fluent.py", line 817, in log_artifacts
    MlflowClient().log_artifacts(run_id, local_dir, artifact_path)
  File "C:\Users\Pinaki\anaconda3\envs\kidney\lib\site-packages\mlflow\tracking\client.py", line 1069, in log_artifacts
    self._tracking_client.log_artifacts(run_id, local_dir, artifact_path)
  File "C:\Users\Pinaki\anaconda3\envs\kidney\lib\site-packages\mlflow\tracking\_tracking_service\client.py", line 448, in log_artifacts
    self._get_artifact_repo(run_id).log_artifacts(local_dir, artifact_path)
  File "C:\Users\Pinaki\anaconda3\envs\kidney\lib\site-packages\mlflow\store\artifact\local_artifact_repo.py", line 58, in log_artifacts
    dir_util.copy_tree(src=local_dir, dst=artifact_dir, preserve_mode=0, preserve_times=0)
  File "C:\Users\Pinaki\anaconda3\envs\kidney\lib\site-packages\setuptools\_distutils\dir_util.py", line 155, in copy_tree
    return list(itertools.chain.from_iterable(map(copy_one, names)))
  File "C:\Users\Pinaki\anaconda3\envs\kidney\lib\site-packages\setuptools\_distutils\dir_util.py", line 197, in _copy_one
    file_util.copy_file(
  File "C:\Users\Pinaki\anaconda3\envs\kidney\lib\site-packages\setuptools\_distutils\file_util.py", line 104, in copy_file
    from distutils._modified import newer
ModuleNotFoundError: No module named 'distutils._modified'
[2024-10-27 12:12:15,660: INFO: main: >>>>>> stage Data Ingestion Stage started <<<<<<]
[2024-10-27 12:12:15,663: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-10-27 12:12:15,665: INFO: common: yaml file: params.yaml loaded successfully]
[2024-10-27 12:12:15,666: INFO: common: created directory at: artifacts]
[2024-10-27 12:12:15,666: INFO: common: created directory at: artifacts/data_ingestion]
[2024-10-27 12:12:15,667: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1Q0WzSl389ofCKy_AX0UjFnQwB73OW-xU/view?usp=drive_link into file artifacts/data_ingestion/data.zip]
[2024-10-27 12:12:51,965: INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/1Q0WzSl389ofCKy_AX0UjFnQwB73OW-xU/view?usp=drive_link into file artifacts/data_ingestion/data.zip]
[2024-10-27 12:12:53,166: INFO: main: >>>>>> stage Data Ingestion Stage completed <<<<<<

x======x]
[2024-10-27 12:12:53,167: INFO: main: >>>>>> stage Prepare base model started <<<<<<]
[2024-10-27 12:12:53,170: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-10-27 12:12:53,172: INFO: common: yaml file: params.yaml loaded successfully]
[2024-10-27 12:12:53,173: INFO: common: created directory at: artifacts]
[2024-10-27 12:12:53,173: INFO: common: created directory at: artifacts/prepare_base_model]
[2024-10-27 12:12:53,808: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-10-27 12:12:54,118: INFO: main: >>>>>> stage Prepare base model completed <<<<<<

x======x]
[2024-10-27 12:12:54,119: INFO: main: >>>>>> stage Training started <<<<<<]
[2024-10-27 12:12:54,122: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-10-27 12:12:54,124: INFO: common: yaml file: params.yaml loaded successfully]
[2024-10-27 12:12:54,125: INFO: common: created directory at: artifacts]
[2024-10-27 12:12:54,126: INFO: common: created directory at: artifacts\training]
[2024-10-27 12:15:32,751: INFO: main: >>>>>> stage Training completed <<<<<<

x======x]
[2024-10-27 12:15:32,751: INFO: main: *******************]
[2024-10-27 12:15:32,751: INFO: main: >>>>>> stage Evaluation Stage started <<<<<<]
[2024-10-27 12:15:32,751: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-10-27 12:15:32,765: INFO: common: yaml file: params.yaml loaded successfully]
[2024-10-27 12:15:32,766: INFO: common: created directory at: artifacts]
[2024-10-27 12:16:19,694: INFO: common: json file saved at: scores.json]
[2024-10-27 12:16:21,835: WARNING: save: Found untraced functions such as _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op while saving (showing 5 of 14). These functions will not be directly callable after loading.]
[2024-10-27 12:16:22,538: INFO: builder_impl: Assets written to: C:\Users\Pinaki\AppData\Local\Temp\tmpn2q9_4f3\model\data\model\assets]
[2024-10-27 12:16:34,434: ERROR: main: No module named 'distutils._modified']
Traceback (most recent call last):
  File "d:/FULL Projects/Kidney-Disease-Classification/main.py", line 54, in <module>
    obj.main()
  File "d:\full projects\kidney-disease-classification\src\cnnClassifier\pipeline\stage_04_model_evaluation.py", line 16, in main
    evaluation.log_into_mlflow()
  File "d:\full projects\kidney-disease-classification\src\cnnClassifier\components\model_evaluation_mlflow.py", line 73, in log_into_mlflow
    mlflow.keras.log_model(self.model, "model")
  File "C:\Users\Pinaki\anaconda3\envs\kidney\lib\site-packages\mlflow\tensorflow\__init__.py", line 222, in log_model
    return Model.log(
  File "C:\Users\Pinaki\anaconda3\envs\kidney\lib\site-packages\mlflow\models\model.py", line 552, in log
    mlflow.tracking.fluent.log_artifacts(local_path, mlflow_model.artifact_path)
  File "C:\Users\Pinaki\anaconda3\envs\kidney\lib\site-packages\mlflow\tracking\fluent.py", line 817, in log_artifacts
    MlflowClient().log_artifacts(run_id, local_dir, artifact_path)
  File "C:\Users\Pinaki\anaconda3\envs\kidney\lib\site-packages\mlflow\tracking\client.py", line 1069, in log_artifacts
    self._tracking_client.log_artifacts(run_id, local_dir, artifact_path)
  File "C:\Users\Pinaki\anaconda3\envs\kidney\lib\site-packages\mlflow\tracking\_tracking_service\client.py", line 448, in log_artifacts
    self._get_artifact_repo(run_id).log_artifacts(local_dir, artifact_path)
  File "C:\Users\Pinaki\anaconda3\envs\kidney\lib\site-packages\mlflow\store\artifact\local_artifact_repo.py", line 58, in log_artifacts
    dir_util.copy_tree(src=local_dir, dst=artifact_dir, preserve_mode=0, preserve_times=0)
  File "C:\Users\Pinaki\anaconda3\envs\kidney\lib\site-packages\setuptools\_distutils\dir_util.py", line 155, in copy_tree
    return list(itertools.chain.from_iterable(map(copy_one, names)))
  File "C:\Users\Pinaki\anaconda3\envs\kidney\lib\site-packages\setuptools\_distutils\dir_util.py", line 197, in _copy_one
    file_util.copy_file(
  File "C:\Users\Pinaki\anaconda3\envs\kidney\lib\site-packages\setuptools\_distutils\file_util.py", line 104, in copy_file
    from distutils._modified import newer
ModuleNotFoundError: No module named 'distutils._modified'
[2024-10-27 12:19:55,537: INFO: main: >>>>>> stage Data Ingestion Stage started <<<<<<]
[2024-10-27 12:19:55,537: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-10-27 12:19:55,537: INFO: common: yaml file: params.yaml loaded successfully]
[2024-10-27 12:19:55,537: INFO: common: created directory at: artifacts]
[2024-10-27 12:19:55,537: INFO: common: created directory at: artifacts/data_ingestion]
[2024-10-27 12:19:55,553: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1Q0WzSl389ofCKy_AX0UjFnQwB73OW-xU/view?usp=drive_link into file artifacts/data_ingestion/data.zip]
[2024-10-27 12:20:28,889: INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/1Q0WzSl389ofCKy_AX0UjFnQwB73OW-xU/view?usp=drive_link into file artifacts/data_ingestion/data.zip]
[2024-10-27 12:20:30,001: INFO: main: >>>>>> stage Data Ingestion Stage completed <<<<<<

x======x]
[2024-10-27 12:20:30,002: INFO: main: >>>>>> stage Prepare base model started <<<<<<]
[2024-10-27 12:20:30,005: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-10-27 12:20:30,008: INFO: common: yaml file: params.yaml loaded successfully]
[2024-10-27 12:20:30,009: INFO: common: created directory at: artifacts]
[2024-10-27 12:20:30,010: INFO: common: created directory at: artifacts/prepare_base_model]
[2024-10-27 12:20:30,624: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-10-27 12:20:30,910: INFO: main: >>>>>> stage Prepare base model completed <<<<<<

x======x]
[2024-10-27 12:20:30,911: INFO: main: >>>>>> stage Training started <<<<<<]
[2024-10-27 12:20:30,914: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-10-27 12:20:30,917: INFO: common: yaml file: params.yaml loaded successfully]
[2024-10-27 12:20:30,918: INFO: common: created directory at: artifacts]
[2024-10-27 12:20:30,918: INFO: common: created directory at: artifacts\training]
[2024-10-27 12:23:08,082: INFO: main: >>>>>> stage Training completed <<<<<<

x======x]
[2024-10-27 12:23:08,083: INFO: main: *******************]
[2024-10-27 12:23:08,083: INFO: main: >>>>>> stage Evaluation Stage started <<<<<<]
[2024-10-27 12:23:08,088: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-10-27 12:23:08,092: INFO: common: yaml file: params.yaml loaded successfully]
[2024-10-27 12:23:08,093: INFO: common: created directory at: artifacts]
[2024-10-27 12:23:55,851: INFO: common: json file saved at: scores.json]
[2024-10-27 12:23:55,852: INFO: common: json file saved at: scores.json]
[2024-10-27 12:23:55,853: INFO: main: >>>>>> stage Evaluation Stage completed <<<<<<

x==========x]
[2024-10-27 12:34:14,073: INFO: stage_04_model_evaluation: >>>>>> stage Evaluation Stage started <<<<<<]
[2024-10-27 12:34:14,073: ERROR: stage_04_model_evaluation: __init__() missing 1 required positional argument: 'config']
Traceback (most recent call last):
  File "d:/FULL Projects/Kidney-Disease-Classification/src/cnnClassifier/pipeline/stage_04_model_evaluation.py", line 23, in <module>
    obj = Evaluation()
TypeError: __init__() missing 1 required positional argument: 'config'
[2024-10-27 12:35:41,112: INFO: main: >>>>>> stage Data Ingestion Stage started <<<<<<]
[2024-10-27 12:35:41,115: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-10-27 12:35:41,116: INFO: common: yaml file: params.yaml loaded successfully]
[2024-10-27 12:35:41,117: INFO: common: created directory at: artifacts]
[2024-10-27 12:35:41,117: INFO: common: created directory at: artifacts/data_ingestion]
[2024-10-27 12:35:41,118: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1Q0WzSl389ofCKy_AX0UjFnQwB73OW-xU/view?usp=drive_link into file artifacts/data_ingestion/data.zip]
[2024-10-27 12:36:13,953: INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/1Q0WzSl389ofCKy_AX0UjFnQwB73OW-xU/view?usp=drive_link into file artifacts/data_ingestion/data.zip]
[2024-10-27 12:36:15,077: INFO: main: >>>>>> stage Data Ingestion Stage completed <<<<<<

x======x]
[2024-10-27 12:36:15,078: INFO: main: >>>>>> stage Prepare base model started <<<<<<]
[2024-10-27 12:36:15,081: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-10-27 12:36:15,083: INFO: common: yaml file: params.yaml loaded successfully]
[2024-10-27 12:36:15,084: INFO: common: created directory at: artifacts]
[2024-10-27 12:36:15,085: INFO: common: created directory at: artifacts/prepare_base_model]
[2024-10-27 12:36:15,685: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-10-27 12:36:16,022: INFO: main: >>>>>> stage Prepare base model completed <<<<<<

x======x]
[2024-10-27 12:36:16,023: INFO: main: >>>>>> stage Training started <<<<<<]
[2024-10-27 12:36:16,026: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-10-27 12:36:16,028: INFO: common: yaml file: params.yaml loaded successfully]
[2024-10-27 12:36:16,028: INFO: common: created directory at: artifacts]
[2024-10-27 12:36:16,029: INFO: common: created directory at: artifacts\training]
[2024-10-27 12:38:56,181: INFO: main: >>>>>> stage Training completed <<<<<<

x======x]
[2024-10-27 12:38:56,182: INFO: main: *******************]
[2024-10-27 12:38:56,182: INFO: main: >>>>>> stage Evaluation Stage started <<<<<<]
[2024-10-27 12:38:56,188: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-10-27 12:38:56,192: INFO: common: yaml file: params.yaml loaded successfully]
[2024-10-27 12:38:56,193: INFO: common: created directory at: artifacts]
[2024-10-27 12:39:43,677: INFO: common: json file saved at: scores.json]
[2024-10-27 12:39:43,678: INFO: common: json file saved at: scores.json]
[2024-10-27 12:39:45,940: WARNING: save: Found untraced functions such as _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op while saving (showing 5 of 14). These functions will not be directly callable after loading.]
[2024-10-27 12:39:46,673: INFO: builder_impl: Assets written to: C:\Users\Pinaki\AppData\Local\Temp\tmpbmseukwq\model\data\model\assets]
[2024-10-27 12:39:58,445: ERROR: main: No module named 'distutils._modified']
Traceback (most recent call last):
  File "d:/FULL Projects/Kidney-Disease-Classification/main.py", line 54, in <module>
    obj.main()
  File "d:\full projects\kidney-disease-classification\src\cnnClassifier\pipeline\stage_04_model_evaluation.py", line 17, in main
    evaluation.log_into_mlflow()
  File "d:\full projects\kidney-disease-classification\src\cnnClassifier\components\model_evaluation_mlflow.py", line 73, in log_into_mlflow
    mlflow.keras.log_model(self.model, "model")
  File "C:\Users\Pinaki\anaconda3\envs\kidney\lib\site-packages\mlflow\tensorflow\__init__.py", line 222, in log_model
    return Model.log(
  File "C:\Users\Pinaki\anaconda3\envs\kidney\lib\site-packages\mlflow\models\model.py", line 552, in log
    mlflow.tracking.fluent.log_artifacts(local_path, mlflow_model.artifact_path)
  File "C:\Users\Pinaki\anaconda3\envs\kidney\lib\site-packages\mlflow\tracking\fluent.py", line 817, in log_artifacts
    MlflowClient().log_artifacts(run_id, local_dir, artifact_path)
  File "C:\Users\Pinaki\anaconda3\envs\kidney\lib\site-packages\mlflow\tracking\client.py", line 1069, in log_artifacts
    self._tracking_client.log_artifacts(run_id, local_dir, artifact_path)
  File "C:\Users\Pinaki\anaconda3\envs\kidney\lib\site-packages\mlflow\tracking\_tracking_service\client.py", line 448, in log_artifacts
    self._get_artifact_repo(run_id).log_artifacts(local_dir, artifact_path)
  File "C:\Users\Pinaki\anaconda3\envs\kidney\lib\site-packages\mlflow\store\artifact\local_artifact_repo.py", line 58, in log_artifacts
    dir_util.copy_tree(src=local_dir, dst=artifact_dir, preserve_mode=0, preserve_times=0)
  File "C:\Users\Pinaki\anaconda3\envs\kidney\lib\site-packages\setuptools\_distutils\dir_util.py", line 155, in copy_tree
    return list(itertools.chain.from_iterable(map(copy_one, names)))
  File "C:\Users\Pinaki\anaconda3\envs\kidney\lib\site-packages\setuptools\_distutils\dir_util.py", line 197, in _copy_one
    file_util.copy_file(
  File "C:\Users\Pinaki\anaconda3\envs\kidney\lib\site-packages\setuptools\_distutils\file_util.py", line 104, in copy_file
    from distutils._modified import newer
ModuleNotFoundError: No module named 'distutils._modified'
[2024-10-27 23:48:00,803: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2024-10-27 23:48:00,836: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-10-27 23:48:00,849: INFO: common: yaml file: params.yaml loaded successfully]
[2024-10-27 23:48:00,849: INFO: common: created directory at: artifacts]
[2024-10-27 23:48:00,849: INFO: common: created directory at: artifacts/data_ingestion]
[2024-10-27 23:48:00,849: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1Q0WzSl389ofCKy_AX0UjFnQwB73OW-xU/view?usp=drive_link into file artifacts/data_ingestion/data.zip]
[2024-10-27 23:48:34,420: INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/1Q0WzSl389ofCKy_AX0UjFnQwB73OW-xU/view?usp=drive_link into file artifacts/data_ingestion/data.zip]
[2024-10-27 23:48:36,133: INFO: main: >>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2024-10-27 23:48:36,135: INFO: main: *******************]
[2024-10-27 23:48:36,136: INFO: main: >>>>>> stage Prepare base model started <<<<<<]
[2024-10-27 23:48:36,136: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-10-27 23:48:36,136: INFO: common: yaml file: params.yaml loaded successfully]
[2024-10-27 23:48:36,146: INFO: common: created directory at: artifacts]
[2024-10-27 23:48:36,146: INFO: common: created directory at: artifacts/prepare_base_model]
[2024-10-27 23:48:37,518: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-10-27 23:48:38,006: INFO: main: >>>>>> stage Prepare base model completed <<<<<<

x==========x]
[2024-10-27 23:48:38,006: INFO: main: *******************]
[2024-10-27 23:48:38,012: INFO: main: >>>>>> stage Training started <<<<<<]
[2024-10-27 23:48:38,017: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-10-27 23:48:38,020: INFO: common: yaml file: params.yaml loaded successfully]
[2024-10-27 23:48:38,020: INFO: common: created directory at: artifacts]
[2024-10-27 23:48:38,022: INFO: common: created directory at: artifacts\training]
[2024-10-27 23:51:26,453: INFO: main: >>>>>> stage Training completed <<<<<<

x==========x]
[2024-10-27 23:51:26,453: INFO: main: *******************]
[2024-10-27 23:51:26,461: INFO: main: >>>>>> stage Evaluation stage started <<<<<<]
[2024-10-27 23:51:26,462: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-10-27 23:51:26,476: INFO: common: yaml file: params.yaml loaded successfully]
[2024-10-27 23:51:26,476: INFO: common: created directory at: artifacts]
[2024-10-27 23:52:20,683: INFO: common: json file saved at: scores.json]
[2024-10-27 23:52:20,683: INFO: common: json file saved at: scores.json]
[2024-10-27 23:52:24,472: WARNING: save: Found untraced functions such as _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op while saving (showing 5 of 14). These functions will not be directly callable after loading.]
[2024-10-27 23:52:25,476: INFO: builder_impl: Assets written to: C:\Users\Pinaki\AppData\Local\Temp\tmp0sy5v6jj\model\data\model\assets]
[2024-10-27 23:52:42,845: ERROR: main: No module named 'distutils._modified']
Traceback (most recent call last):
  File "d:/FULL Projects/Kidney-Disease-Classification/main.py", line 56, in <module>
    model_evalution.main()
  File "d:\full projects\kidney-disease-classification\src\cnnClassifier\pipeline\stage_04_model_evaluation.py", line 17, in main
    evaluation.log_into_mlflow()
  File "d:\full projects\kidney-disease-classification\src\cnnClassifier\components\model_evaluation_mlflow.py", line 73, in log_into_mlflow
    mlflow.keras.log_model(self.model, "model")
  File "C:\Users\Pinaki\anaconda3\envs\kidney\lib\site-packages\mlflow\tensorflow\__init__.py", line 222, in log_model
    return Model.log(
  File "C:\Users\Pinaki\anaconda3\envs\kidney\lib\site-packages\mlflow\models\model.py", line 552, in log
    mlflow.tracking.fluent.log_artifacts(local_path, mlflow_model.artifact_path)
  File "C:\Users\Pinaki\anaconda3\envs\kidney\lib\site-packages\mlflow\tracking\fluent.py", line 817, in log_artifacts
    MlflowClient().log_artifacts(run_id, local_dir, artifact_path)
  File "C:\Users\Pinaki\anaconda3\envs\kidney\lib\site-packages\mlflow\tracking\client.py", line 1069, in log_artifacts
    self._tracking_client.log_artifacts(run_id, local_dir, artifact_path)
  File "C:\Users\Pinaki\anaconda3\envs\kidney\lib\site-packages\mlflow\tracking\_tracking_service\client.py", line 448, in log_artifacts
    self._get_artifact_repo(run_id).log_artifacts(local_dir, artifact_path)
  File "C:\Users\Pinaki\anaconda3\envs\kidney\lib\site-packages\mlflow\store\artifact\local_artifact_repo.py", line 58, in log_artifacts
    dir_util.copy_tree(src=local_dir, dst=artifact_dir, preserve_mode=0, preserve_times=0)
  File "C:\Users\Pinaki\anaconda3\envs\kidney\lib\site-packages\setuptools\_distutils\dir_util.py", line 155, in copy_tree
    return list(itertools.chain.from_iterable(map(copy_one, names)))
  File "C:\Users\Pinaki\anaconda3\envs\kidney\lib\site-packages\setuptools\_distutils\dir_util.py", line 197, in _copy_one
    file_util.copy_file(
  File "C:\Users\Pinaki\anaconda3\envs\kidney\lib\site-packages\setuptools\_distutils\file_util.py", line 104, in copy_file
    from distutils._modified import newer
ModuleNotFoundError: No module named 'distutils._modified'
[2024-10-28 00:03:38,987: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2024-10-28 00:03:38,987: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-10-28 00:03:38,987: INFO: common: yaml file: params.yaml loaded successfully]
[2024-10-28 00:03:38,987: INFO: common: created directory at: artifacts]
[2024-10-28 00:03:38,987: INFO: common: created directory at: artifacts/data_ingestion]
[2024-10-28 00:03:38,987: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1Q0WzSl389ofCKy_AX0UjFnQwB73OW-xU/view?usp=drive_link into file artifacts/data_ingestion/data.zip]
[2024-10-28 00:04:12,146: INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/1Q0WzSl389ofCKy_AX0UjFnQwB73OW-xU/view?usp=drive_link into file artifacts/data_ingestion/data.zip]
[2024-10-28 00:04:13,818: INFO: main: >>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2024-10-28 00:04:13,818: INFO: main: *******************]
[2024-10-28 00:04:13,818: INFO: main: >>>>>> stage Prepare base model started <<<<<<]
[2024-10-28 00:04:13,835: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-10-28 00:04:13,840: INFO: common: yaml file: params.yaml loaded successfully]
[2024-10-28 00:04:13,842: INFO: common: created directory at: artifacts]
[2024-10-28 00:04:13,842: INFO: common: created directory at: artifacts/prepare_base_model]
[2024-10-28 00:04:14,708: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-10-28 00:04:15,117: INFO: main: >>>>>> stage Prepare base model completed <<<<<<

x==========x]
[2024-10-28 00:04:15,133: INFO: main: *******************]
[2024-10-28 00:04:15,133: INFO: main: >>>>>> stage Training started <<<<<<]
[2024-10-28 00:04:15,133: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-10-28 00:04:15,150: INFO: common: yaml file: params.yaml loaded successfully]
[2024-10-28 00:04:15,151: INFO: common: created directory at: artifacts]
[2024-10-28 00:04:15,151: INFO: common: created directory at: artifacts\training]
[2024-10-28 00:07:05,495: INFO: main: >>>>>> stage Training completed <<<<<<

x==========x]
[2024-10-28 00:07:05,495: INFO: main: *******************]
[2024-10-28 00:07:05,495: INFO: main: >>>>>> stage Evaluation stage started <<<<<<]
[2024-10-28 00:07:05,495: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-10-28 00:07:05,513: INFO: common: yaml file: params.yaml loaded successfully]
[2024-10-28 00:07:05,517: INFO: common: created directory at: artifacts]
[2024-10-28 00:08:04,843: INFO: common: json file saved at: scores.json]
[2024-10-28 00:08:04,843: INFO: common: json file saved at: scores.json]
[2024-10-28 00:08:07,794: WARNING: save: Found untraced functions such as _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op while saving (showing 5 of 14). These functions will not be directly callable after loading.]
[2024-10-28 00:08:08,821: INFO: builder_impl: Assets written to: C:\Users\Pinaki\AppData\Local\Temp\tmpcymn6yuz\model\data\model\assets]
[2024-10-28 00:08:28,351: INFO: main: >>>>>> stage Evaluation stage completed <<<<<<

x==========x]
[2024-10-28 00:16:15,551: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2024-10-28 00:16:15,551: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-10-28 00:16:15,551: INFO: common: yaml file: params.yaml loaded successfully]
[2024-10-28 00:16:15,551: INFO: common: created directory at: artifacts]
[2024-10-28 00:16:15,551: INFO: common: created directory at: artifacts/data_ingestion]
[2024-10-28 00:16:15,551: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1Q0WzSl389ofCKy_AX0UjFnQwB73OW-xU/view?usp=drive_link into file artifacts/data_ingestion/data.zip]
[2024-10-28 00:16:48,614: INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/1Q0WzSl389ofCKy_AX0UjFnQwB73OW-xU/view?usp=drive_link into file artifacts/data_ingestion/data.zip]
[2024-10-28 00:16:50,758: INFO: main: >>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2024-10-28 00:16:50,758: INFO: main: *******************]
[2024-10-28 00:16:50,758: INFO: main: >>>>>> stage Prepare base model started <<<<<<]
[2024-10-28 00:16:50,774: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-10-28 00:16:50,774: INFO: common: yaml file: params.yaml loaded successfully]
[2024-10-28 00:16:50,774: INFO: common: created directory at: artifacts]
[2024-10-28 00:16:50,782: INFO: common: created directory at: artifacts/prepare_base_model]
[2024-10-28 00:16:51,757: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-10-28 00:16:52,293: INFO: main: >>>>>> stage Prepare base model completed <<<<<<

x==========x]
[2024-10-28 00:16:52,293: INFO: main: *******************]
[2024-10-28 00:16:52,293: INFO: main: >>>>>> stage Training started <<<<<<]
[2024-10-28 00:16:52,304: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-10-28 00:16:52,311: INFO: common: yaml file: params.yaml loaded successfully]
[2024-10-28 00:16:52,313: INFO: common: created directory at: artifacts]
[2024-10-28 00:16:52,313: INFO: common: created directory at: artifacts\training]
[2024-10-28 00:19:47,902: INFO: main: >>>>>> stage Training completed <<<<<<

x==========x]
[2024-10-28 00:19:47,902: INFO: main: *******************]
[2024-10-28 00:19:47,902: INFO: main: >>>>>> stage Evaluation stage started <<<<<<]
[2024-10-28 00:19:47,919: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-10-28 00:19:47,934: INFO: common: yaml file: params.yaml loaded successfully]
[2024-10-28 00:19:47,935: INFO: common: created directory at: artifacts]
[2024-10-28 00:20:44,653: INFO: common: json file saved at: scores.json]
[2024-10-28 00:20:44,653: INFO: common: json file saved at: scores.json]
[2024-10-28 00:20:47,936: WARNING: save: Found untraced functions such as _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op while saving (showing 5 of 14). These functions will not be directly callable after loading.]
[2024-10-28 00:20:48,983: INFO: builder_impl: Assets written to: C:\Users\Pinaki\AppData\Local\Temp\tmp5qwz25bo\model\data\model\assets]
[2024-10-28 00:21:07,679: INFO: main: >>>>>> stage Evaluation stage completed <<<<<<

x==========x]
[2024-10-28 00:27:52,472: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-10-28 00:27:52,472: INFO: common: yaml file: params.yaml loaded successfully]
[2024-10-28 00:27:52,488: INFO: common: created directory at: artifacts]
[2024-10-28 00:27:52,488: INFO: common: created directory at: artifacts/data_ingestion]
[2024-10-28 00:27:52,488: INFO: 986331348: Downloading data from https://drive.google.com/file/d/1Q0WzSl389ofCKy_AX0UjFnQwB73OW-xU/view?usp=drive_link into file artifacts/data_ingestion/data.zip]
[2024-10-28 00:28:27,857: INFO: 986331348: Downloaded data from https://drive.google.com/file/d/1Q0WzSl389ofCKy_AX0UjFnQwB73OW-xU/view?usp=drive_link into file artifacts/data_ingestion/data.zip]
[2024-10-28 00:29:03,646: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-10-28 00:29:03,686: INFO: common: yaml file: params.yaml loaded successfully]
[2024-10-28 00:29:03,703: INFO: common: created directory at: artifacts]
[2024-10-28 00:29:03,719: INFO: common: created directory at: artifacts/prepare_base_model]
[2024-10-28 00:29:05,126: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-10-28 00:29:40,354: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-10-28 00:29:40,362: INFO: common: yaml file: params.yaml loaded successfully]
[2024-10-28 00:29:40,364: INFO: common: created directory at: artifacts]
[2024-10-28 00:29:40,368: INFO: common: created directory at: artifacts\training]
[2024-10-28 00:34:09,194: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-10-28 00:34:09,194: INFO: common: yaml file: params.yaml loaded successfully]
[2024-10-28 00:34:09,194: INFO: common: created directory at: artifacts]
[2024-10-28 00:35:05,170: INFO: common: json file saved at: scores.json]
[2024-10-28 00:35:08,044: WARNING: save: Found untraced functions such as _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op while saving (showing 5 of 14). These functions will not be directly callable after loading.]
[2024-10-28 00:35:08,843: INFO: builder_impl: Assets written to: C:\Users\Pinaki\AppData\Local\Temp\tmpjc336ghu\model\data\model\assets]
[2024-10-28 01:20:14,649: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2024-10-28 01:20:14,656: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-10-28 01:20:14,656: INFO: common: yaml file: params.yaml loaded successfully]
[2024-10-28 01:20:14,656: INFO: common: created directory at: artifacts]
[2024-10-28 01:20:14,656: INFO: common: created directory at: artifacts/data_ingestion]
[2024-10-28 01:20:14,656: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1Q0WzSl389ofCKy_AX0UjFnQwB73OW-xU/view?usp=drive_link into file artifacts/data_ingestion/data.zip]
[2024-10-28 01:20:46,851: INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/1Q0WzSl389ofCKy_AX0UjFnQwB73OW-xU/view?usp=drive_link into file artifacts/data_ingestion/data.zip]
[2024-10-28 01:20:47,912: INFO: main: >>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2024-10-28 01:20:47,912: INFO: main: *******************]
[2024-10-28 01:20:47,912: INFO: main: >>>>>> stage Prepare base model started <<<<<<]
[2024-10-28 01:20:47,928: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-10-28 01:20:47,935: INFO: common: yaml file: params.yaml loaded successfully]
[2024-10-28 01:20:47,936: INFO: common: created directory at: artifacts]
[2024-10-28 01:20:47,936: INFO: common: created directory at: artifacts/prepare_base_model]
[2024-10-28 01:20:48,544: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-10-28 01:20:48,840: INFO: main: >>>>>> stage Prepare base model completed <<<<<<

x==========x]
[2024-10-28 01:20:48,841: INFO: main: *******************]
[2024-10-28 01:20:48,841: INFO: main: >>>>>> stage Training started <<<<<<]
[2024-10-28 01:20:48,844: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-10-28 01:20:48,847: INFO: common: yaml file: params.yaml loaded successfully]
[2024-10-28 01:20:48,848: INFO: common: created directory at: artifacts]
[2024-10-28 01:20:48,849: INFO: common: created directory at: artifacts\training]
[2024-10-28 01:23:22,831: INFO: main: >>>>>> stage Training completed <<<<<<

x==========x]
[2024-10-28 01:23:22,832: INFO: main: *******************]
[2024-10-28 01:23:22,833: INFO: main: >>>>>> stage Evaluation stage started <<<<<<]
[2024-10-28 01:23:22,837: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-10-28 01:23:22,841: INFO: common: yaml file: params.yaml loaded successfully]
[2024-10-28 01:23:22,842: INFO: common: created directory at: artifacts]
[2024-10-28 01:24:11,283: INFO: common: json file saved at: scores.json]
[2024-10-28 01:24:11,283: INFO: common: json file saved at: scores.json]
[2024-10-28 01:24:12,127: INFO: _client: HTTP Request: GET https://dagshub.com/api/v1/user "HTTP/1.1 200 OK"]
[2024-10-28 01:24:12,127: INFO: helpers: Accessing as Luminousbyte]
[2024-10-28 01:24:12,962: INFO: _client: HTTP Request: GET https://dagshub.com/api/v1/repos/Luminousbyte/Kidney-Disease-Classification "HTTP/1.1 200 OK"]
[2024-10-28 01:24:12,962: INFO: helpers: Initialized MLflow to track repo "Luminousbyte/Kidney-Disease-Classification"]
[2024-10-28 01:24:12,979: INFO: helpers: Repository Luminousbyte/Kidney-Disease-Classification initialized!]
[2024-10-28 01:24:16,997: WARNING: save: Found untraced functions such as _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op while saving (showing 5 of 14). These functions will not be directly callable after loading.]
[2024-10-28 01:24:17,706: INFO: builder_impl: Assets written to: C:\Users\Pinaki\AppData\Local\Temp\tmpxwycn8zm\model\data\model\assets]
[2024-10-28 01:24:55,192: INFO: main: >>>>>> stage Evaluation stage completed <<<<<<

x==========x]
[2024-10-28 01:31:15,393: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2024-10-28 01:31:15,393: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-10-28 01:31:15,409: INFO: common: yaml file: params.yaml loaded successfully]
[2024-10-28 01:31:15,409: INFO: common: created directory at: artifacts]
[2024-10-28 01:31:15,409: INFO: common: created directory at: artifacts/data_ingestion]
[2024-10-28 01:31:15,409: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1Q0WzSl389ofCKy_AX0UjFnQwB73OW-xU/view?usp=drive_link into file artifacts/data_ingestion/data.zip]
[2024-10-28 01:31:48,562: INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/1Q0WzSl389ofCKy_AX0UjFnQwB73OW-xU/view?usp=drive_link into file artifacts/data_ingestion/data.zip]
[2024-10-28 01:31:49,525: INFO: main: >>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2024-10-28 01:31:49,526: INFO: main: *******************]
[2024-10-28 01:31:49,526: INFO: main: >>>>>> stage Prepare base model started <<<<<<]
[2024-10-28 01:31:49,529: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-10-28 01:31:49,531: INFO: common: yaml file: params.yaml loaded successfully]
[2024-10-28 01:31:49,532: INFO: common: created directory at: artifacts]
[2024-10-28 01:31:49,533: INFO: common: created directory at: artifacts/prepare_base_model]
[2024-10-28 01:31:50,081: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-10-28 01:31:50,366: INFO: main: >>>>>> stage Prepare base model completed <<<<<<

x==========x]
[2024-10-28 01:31:50,366: INFO: main: *******************]
[2024-10-28 01:31:50,366: INFO: main: >>>>>> stage Training started <<<<<<]
[2024-10-28 01:31:50,382: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-10-28 01:31:50,382: INFO: common: yaml file: params.yaml loaded successfully]
[2024-10-28 01:31:50,382: INFO: common: created directory at: artifacts]
[2024-10-28 01:31:50,388: INFO: common: created directory at: artifacts\training]
[2024-10-28 01:38:13,877: INFO: main: >>>>>> stage Training completed <<<<<<

x==========x]
[2024-10-28 01:38:13,878: INFO: main: *******************]
[2024-10-28 01:38:13,879: INFO: main: >>>>>> stage Evaluation stage started <<<<<<]
[2024-10-28 01:38:13,884: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-10-28 01:38:13,887: INFO: common: yaml file: params.yaml loaded successfully]
[2024-10-28 01:38:13,889: INFO: common: created directory at: artifacts]
[2024-10-28 01:39:10,470: INFO: common: json file saved at: scores.json]
[2024-10-28 01:39:10,471: INFO: common: json file saved at: scores.json]
[2024-10-28 01:39:11,397: INFO: _client: HTTP Request: GET https://dagshub.com/api/v1/user "HTTP/1.1 200 OK"]
[2024-10-28 01:39:11,405: INFO: helpers: Accessing as Luminousbyte]
[2024-10-28 01:39:12,730: INFO: _client: HTTP Request: GET https://dagshub.com/api/v1/repos/Luminousbyte/Kidney-Disease-Classification "HTTP/1.1 200 OK"]
[2024-10-28 01:39:12,735: INFO: helpers: Initialized MLflow to track repo "Luminousbyte/Kidney-Disease-Classification"]
[2024-10-28 01:39:12,737: INFO: helpers: Repository Luminousbyte/Kidney-Disease-Classification initialized!]
[2024-10-28 01:39:17,554: WARNING: save: Found untraced functions such as _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op while saving (showing 5 of 14). These functions will not be directly callable after loading.]
[2024-10-28 01:39:18,477: INFO: builder_impl: Assets written to: C:\Users\Pinaki\AppData\Local\Temp\tmp8zpg11mb\model\data\model\assets]
[2024-10-28 01:40:01,779: INFO: main: >>>>>> stage Evaluation stage completed <<<<<<

x==========x]
